### DISCUSSION TITLE
Are there any additional documents (e.g., results/accuracy) I can refer to in my thesis regarding Kaiaulu?

### DISCUSSION THREAD

**Discussion by carlosparadis** [sailuh/kaiaulu#118] [Q&A] [answered]
@tuejari Also moving this here since it may be of help to others. 

>  I do know how to cite Kaiaulu from the Readme, but I do not get any results when I Google the title.  For example, could I mention http://itm0.shidler.hawaii.edu/kaiaulu/index.html as source for documentation? Furthermore, am I allowed to include the diagram you sent me in my thesis? I am not sure whether I am going to include it, as I think it might be too detailed, but if I end up wanting to use it, I would like to know whether that is allowed. I was thinking it could show the progress of Kaiaulu at the time I used it.

**Comment by carlosparadis** [comment, marked as answer]
@maelstromdat also asked me this. For now, your only option is to cite:

```
@phdthesis{Paradis:2021,
  author  = "Carlos Paradis",
  title   = "PERCEIVE: Proactive Exploration of Risky Concept Emergence for Identifying Vulnerabilities \& Exposures",
  school  = "University of Hawaii at Manoa",
  year    = "2021",
  month = "May"
}
```

> I do know how to cite Kaiaulu from the Readme, but I do not get any results when I Google the title.

The reason you can't find it is because of the `year` and `month` fields, or more specifically because I graduated too recently and the thesis has not yet been indexed online! 😄 There will be at least 3 locations you will be able to find it eventually: ProQuest, University of Hawaii's ScholarSpace, and NASA's NTRS system. Unfortunately, none of the 3 processed it yet so it is yet to go public.


>  Furthermore, am I allowed to include the diagram you sent me in my thesis?

You can include the diagram from #117, just attribute it to the dissertation I guess. As you said, I think that is overkill: The diagram does not show **the path** it calculates social smells, but **all possible paths** it could. 

> I am not sure whether I am going to include it, as I think it might be too detailed, but if I end up wanting to use it, I would like to know whether that is allowed. I was thinking it could show the progress of Kaiaulu at the time I used it. 

The actual path you chose is what is in the R notebook, so you could mention it along with the commit hash version if you want to be ultra precise.

I am not sure if http://itm0.shidler.hawaii.edu/kaiaulu will be used forever since it is a local Shidler server, but if you want to reflect the state of Kaiaulu in your dissertation I would include https://github.com/sailuh/kaiaulu and the latest commit hash of the version you used for analysis (plus perhaps the project configuration files in an appendix for full reproducibility). You can note the repository is currently private and it will be public eventually.

**Reply by tuejari** [reply]
Thank you for the explanation! I hadn't thought about using the commit hash, but that is a smart idea!

**Reply by carlosparadis** [reply]
Hopefully in the future when the codebase is more stable we will have formal releases so that is referred to, but for now, where the code is still evolving aggressively the commit hash should do the trick 😄
